cla_public/apps/base/forms.py

```diff
@@ -49,15 +51,21 @@ class FeedbackForm(Honeypot, BabelTranslationsFormMixin, Form):
         validators=[InputRequired()])
 
     def api_payload(self):
+        environment = os.environ.get('CLA_ENV')
```

Comment:
could add this to the main settings file

CLA_ENV = os.environ.get('CLA_ENV')

then

environment = current_app.config['CLA_ENV']